lista03/ex01/ex01/src/ex01/Bicicleta.java

```diff
@@ -0,0 +1,25 @@
+package ex01;
+
+public class Bicicleta extends Veiculo {
+    protected int numMarchas;
+    protected int aroRoda;
```

Comment:
Bicicleta é uma classe "final" não tem porque esses atributos serem protegidos
```suggestion
    private int numMarchas;
    private int aroRoda;
```